packages/harpparts/src/tuning/types/tuning-types.ts

```diff
@@ -62,16 +63,19 @@ export type Tuning = {
   readonly id: TuningIds
   readonly shortName?: string
   readonly category: TuningCategories
-  readonly reedArray: ReedArray
+  readonly reedArrays: readonly [ReedArray, ReedArray]
```

Comment:
It might be beneficial to have some parity between the possibility of multiple reed arrays, and the possibility of multiple harpfaces. Even though they are in different projects I'm sure that would make things clearer.

Reply:
They're not even in different projects are they, it's just that we mostly use the harpfacematrices in the harpguru-core project, but they are defined here.

Reply:
Dealt with in #130 .